kde-misc/ktoshiba/ktoshiba-5.2.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
```

Comment:
in gentoo kde we use one empty line between EAPI declaration the other vars/inherits